catalog/dags/providers/provider_api_scripts/science_museum.py

```diff
@@ -166,9 +167,13 @@ def _get_creator_info(attributes):
     def check_url(url: str | None) -> str | None:
         if not url:
             return None
-        if url.startswith("http"):
-            return url
-        return f"https://coimages.sciencemuseumgroup.org.uk/{url}"
+
+        # Will return None if url 403s
+        return rewrite_redirected_url(
+            url
+            if url.startswith("http")
+            else f"https://coimages.sciencemuseumgroup.org.uk/{url}"
+        )
```

Comment:
Ooo, great call! Although, this may have a side-effect as denoted below.

Reply:
Responded below!